src/utils/test-utils.tsx

```diff
@@ -16,31 +22,42 @@ const goTo = (route?: string, name?: string) => {
     );
 };
 
-const waitForLoading = async () => {
-    return Promise.resolve();
-};
```

Comment:
This was a stand in and not working well. So removing. Just going to stick with each test having it's own wait